__tests__/qraphql_queries/category.test.js

```diff
@@ -0,0 +1,258 @@
+import EasyGraphQLTester from "easygraphql-tester";
```

Comment:
There are some minor problems that make several tests in this file fail (running these tests clearly says what's wrong with them, therefore I'm not gonna point out every single line that causes error). My guess is that newest commits provided changes that weren't taken into account in this test. Please, fix that.

Reply:
Great that you noticed that, I forgot to check the graphql tests after making changes to the schema.